modules/carto/src/layers/carto-binary-tile-layer.js

```diff
@@ -0,0 +1,56 @@
+/* global TextDecoder */
+import {log} from '@deck.gl/core';
+import {ClipExtension} from '@deck.gl/extensions';
+import {MVTLayer} from '@deck.gl/geo-layers';
+import {GeoJsonLayer} from '@deck.gl/layers';
+import {geojsonToBinary} from '@loaders.gl/gis';
+
+function parseJSON(arrayBuffer) {
+  return JSON.parse(new TextDecoder().decode(arrayBuffer));
+}
+
+function parseCartoBinaryTile(arrayBuffer, options) {
+  if (!arrayBuffer) return null;
+  const formatTiles = (options && options.formatTiles) || 'geojson';
+  if (formatTiles === 'geojson') return geojsonToBinary(parseJSON(arrayBuffer).features);
+  log.assert(formatTiles === 'geojson', `formatTiles must be geojson`);
+  return null;
+}
+
+const CartoBinaryTileLoader = {
+  name: 'CARTO Binary Tile',
+  id: 'cartoBinaryTile',
+  module: 'carto',
+  extensions: ['pbf'],
+  mimeTypes: ['application/x-protobuf'],
+  category: 'geometry',
+  worker: false,
+  parse: async (arrayBuffer, options) => parseCartoBinaryTile(arrayBuffer, options),
+  parseSync: parseCartoBinaryTile
+};
+
+// Currently we only support loading via geojson, but in future the data
+// format will be binary, as such keep `binary` in layer name
+export default class CartoBinaryTileLayer extends MVTLayer {
+  renderSubLayers(props) {
+    if (props.data === null) {
+      return null;
+    }
+
+    props.autoHighlight = false;
```

Comment:
It's temporal, but if it's easy would be great if we can warn the user when the autoHighlight is changed to false

Reply:
I'm just mimicing what the MVTLayer already does here, so there is no change in behavior